posthog/models/person/person.py

```diff
@@ -69,14 +69,11 @@ def split_person(self, main_distinct_id: Optional[str]):
 
                 from posthog.models.person.util import create_person, create_person_distinct_id
 
-                create_person_distinct_id(
```

Comment:
How is splitting distinctIDs handled now?

Reply:
Mostly correctly. Note that the previous code didn't actually do anything in this branch for pdi2 table.